daemon/config.go

```diff
@@ -206,6 +212,34 @@ func getConflictFreeConfiguration(configFile string, flags *flag.FlagSet) (*Conf
 			return nil, err
 		}
 
+		// make sure that values set in the config file with nullable values, like `false`,
```

Comment:
Update function getConflictFreeConfiguration comment to say that "flags will be updated, if necessary".